src/org/linphone/InCallActivity.java

```diff
@@ -1842,57 +1842,31 @@ public void onAnimationEnd(Animation animation) {
 	}
 
 	private void hideAnimatedLandscapeCallOptions() {
-		Animation animation = slideOutTopToBottom;
-		if (isTransferAllowed) {
-			animation.setAnimationListener(new AnimationListener() {
-				@Override
-				public void onAnimationStart(Animation animation) {
-				}
+		Animation animation = slideOutLeftToRight;
+		animation.setAnimationListener(new AnimationListener() {
+			@Override
+			public void onAnimationStart(Animation animation) {
+			}
 
-				@Override
-				public void onAnimationRepeat(Animation animation) {
-				}
+			@Override
+			public void onAnimationRepeat(Animation animation) {
+			}
 
-				@Override
-				public void onAnimationEnd(Animation animation) {
-					transfer.setAnimation(null);
+			@Override
+			public void onAnimationEnd(Animation animation) {
+				if (isTransferAllowed) {
 					transfer.setVisibility(View.INVISIBLE);
```

Comment:
if(isTransferIsAllowed); set transfer to invisible? Does this break call transfer?